src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
-
-Mobile friendliness should be then fundamental for any website looking to provide a good search experience -and as a consequence, grow in search results.
-
-A mobile friendly website can be implemented through different configurations: by using a responsive web design, with dynamic serving or via a separate mobile web version, although this last approach is not recommended anymore by Google who endorse responsive web design instead.
-
-#### Viewport Meta Tag
-
-The browser’s viewport is the visible area of a page content, that changes depending on the used device. The "viewport meta tag" allows you to specify to browsers the width and scaling of the viewport, so that it is correctly sized across different devices. Responsive websites use the viewport meta tag as well as CSS media queries to deliver a mobile friendly experience.
-
-When looking at the viewport tag inclusion, it was found that 42.98% of the analyzed mobile home pages and 43.2% desktop ones are featuring it with an `initial-scale=1,width=device-width` content attribute value; but that 10.84% of mobile home pages and 16.18% of desktop ones are not including the tag at all, suggesting that they are not yet mobile friendly.
-
-<figure markdown>
-Viewport | Mobile | Desktop
--- | -- | --
-`initial-scale=1,width=device-width` | 42.98% | 43.20%
-*not-set*  | 10.84% | 16.18%
-`initial-scale=1,maximum-scale=1,width=device-width` | 5.88% | 5.72%
-`initial-scale=1,maximum-scale=1,user-scalable=no,width=device-width` | 5.56% | 4.81%
-`initial-scale=1,maximum-scale=1,user-scalable=0,width=device-width` | 3.93% | 3.73%
-
-<figcaption>{{ figure_link(caption="Viewport meta values.", sheets_gid="479500659", sql_file="../03_Markup/summary_pages_by_device_and_viewport.sql") }}</figcaption>
+This section explores the opportunities for optimization related to web configurations and elements that may not directly affect a site's crawlability or indexability, but have either been confirmed by search engines to be used as ranking signals or will affect the capacity of websites to leverage search features.
+
+### Mobile friendliness
+
+With the increasing popularity of mobile devices to browse and search across the web, search engines have been taking mobile friendliness into consideration as a ranking factor for several years.
+
+{# TODO(authors): MFI has been discussed earlier, so the "in fact" doesn't pack as much punch this time. Consider rephrasing. #}
+In fact, [since 2016](https://developers.google.com/search/blog/2016/11/mobile-first-indexing) Google has been moving to a mobile-first index, meaning that the content that is crawled, indexed, and ranked is the one accessible to mobile users and the [Smartphone Googlebot](https://developers.google.com/search/docs/advanced/crawling/googlebot?hl=en).
+
+{# TODO(authors): Can you clarify the timeline? You say "July last year" but the blog post is dated July 2020. Would "July this year" change how you structure this sentence chronologically? #}
```

Comment:
@aleyda

Reply:
@rviscomi I reworded this too along the previous change.